src/node/db/AuthorManager.js

```diff
@@ -107,6 +107,10 @@ exports.doesAuthorExists = exports.doesAuthorExist;
  * @param {String} token The token
  */
 exports.getAuthor4Token = async (token) => {
+  // e.g. in case of an export of a public pad, a token is not needed
+  if (token == null) {
+    return;
+  }
```

Comment:
I don't think the comment is necessary, and I would explicitly return `null`:

```suggestion
  if (token == null) return null;
```